feat(secret): describe the syncing behavior of secret (#2206)
kumahq/kuma-website: app/_src/production/secure-deployment/secrets.md

{% endtab %}
{% endtabs %}

## Behaviour in multi-zone deployment

> The Secrets can be synced from global to zones, not the other way around.

Reviewer: ```suggestion
The Secrets are synced from global to zones, not the other way around.
```

{% if_version lte:2.9.x %}

> It's invalid to create Secrets on zone, they would be automatically deleted further.

Reviewer: ```suggestion
It's invalid to create Secrets on zone, if you do they are deleted by the CP.
```

{% endif_version %}
{% if_version gte:2.10.x %}

> You can also create Secrets on zone but if there's a name conflict by the Secret which will be created on Global, then the syncing from global to zones would be ignored.

Reviewer: ```suggestion
If there's a name conflict with a Secret on Global, then the secret will not be synced and a warning will appear in the logs.
```
